docs/userguides/cache.md

```diff
@@ -0,0 +1,74 @@
+# Cache
+
+Use the cache plugin to store provider data in an sqlite database.
+
+```bash
+ape cache init --network <your-choice-of-network>
+```
+
+You will need to set-up your network connection that you want to work off of.
+
+For example, if you want to work off of the infura provider, you have to install the infura plugin.
+
+```bash
+ape plugins install infura
+```
+
+Then, to connect to the mainnet:
+
+```bash
+ape cache init --network ethereum:mainnet:infura
+```
+
+This creates a SQLite database file in the hidden ape folder.
+
+## Get data from the provider.
+
+Use `ape console`:
+
+```bash
+ape console --network ethereum:mainnet:infura
+```
+
+Run a few queries:
+
+```python
+chain.blocks.query("*", stop_block=20)
+```
+
+```python
```

Comment:
Could this be combined with the code block above? and possibly include IPython output?